src/blocks/remote-data-container/components/popovers/InputPopover.tsx

```diff
@@ -39,7 +39,7 @@ export function InputPopover( props: InputPopoverProps ) {
 	function onSelectItem(): void {
 		onSelect( inputState );
 		close();
-		sendTracksEvent( 'remotedatablocks_add_block', {
+		sendTracksEvent( 'add_block', {
```

Comment:
In case we have an internal list of these events, we should update that with these new names. Guessing @smithjw1 might have something like that?